src/App.js

```diff
@@ -30,6 +31,7 @@ function App() {
         <Stack sx={{ minHeight: '100vh' }}>
           <CssBaseline />
           <TopMenu />
+          <ScrollToHashElement />
```

Comment:
Put it under the line with <BrowserRouter> (here it seems to have some layout purpose)

Reply:
@Zuzana-bor  Sorry, the word BrowserRouter was formatted as HTML and hidden 😿  Please move the line again.

Reply:
```
<ThemeProvider theme={theme}>
    <BrowserRouter>
      <ScrollToHashElement />
  ```